core/api/pod-api/src/api.ts

```diff
@@ -101,7 +102,10 @@ export interface PolyIn {
      */
     has(...quads: RDF.Quad[]): Promise<boolean>;
 }
-
+ export interface Entry {
```

Comment:
Also an empty line in front of this please, so between the `}` above and this line.